docs/expressions/expressions-javascript.md

```diff
@@ -85,6 +85,17 @@ Then you can access the `"Title"` field like this:
 
 > If your activity is a direct child of an HTTP Endpoint activity, you can access its output directly via the `input` variable, which will be an instance of `HttpRequestModel`.
 
+#### SendHttpRequest Activity
+
+The output property of the **SendHttpRequest** has two output properties:
```

Comment:
Please change this sentence to:

`The **SendHttpRequest** activity has two output properties:`